docs/UserGuide.md

```diff
@@ -171,12 +171,63 @@ Examples:
 
 ### Appointment Commands
 
+An **appointment** is defined by several fields: an appointment type, appointment date & time, ID of the patient who is associated with the appointment, sickness, and medicine.  You can interact with appointments in DocTrack using the instructions below:
+
 <br>
 
-TODO: Add appointment commands here.
+##### Adding an appointment: `add appt`
+
+Adds an appointment to the DocTrack system. 
+
+Format: `add appt i/PATIENT_ID d/DATE_TIME ty/APPOINTMENT_TYPE s/SICKNESS m/MEDICINE`
+
+Examples:
 
+- `add appt i/John Doe d/2024-12-01 09:30 ty/Consulation s/Diabetes m/Insulin`
+- `add appt i/Jo An d/30/01/2024-12-01 16:40:00 ty/Follow-up s/Asthma m/Inhaler`
 ---
 
+<br>
+
+##### Listing all persons : `list appt`
+
+Shows a list of all appointments in the DocTrack system.
+
+Format: `list appt`
+
+<br>
+
+##### Editing a person : `edit appt`
+
+Edits an existing appointment in the system.
+
+Format: `edit appt INDEX [i/Jo An] [d/2024-12-01 11:30] [ty/Screening] [s/Asthma] [m/Inhaler]`
```

Comment:
For the format should we put as the parameters rather than a specific example? Eg `edit appt INDEX [i/PATIENT_ID] [d/DATE_TIME] [ty/APPOINTMENT_TYPE] [s/SICKNESS] [m/MEDICINE]`?